provider/coredns_test.go

```diff
@@ -311,6 +329,7 @@ func applyServiceChanges(provider coreDNSProvider, changes *plan.Changes) {
 }
 
 func validateServices(services, expectedServices map[string]*Service, t *testing.T, step int) {
+	t.Helper()
```

Comment:
What is this line doing?

Reply:
@Raffo without `t.Helper()` the stackrace points to the line in the function instead of the specific failing test where this function was used. So adding `t.Helper()` makes it much easier to understand which tests is exactly failing during the test run